templates/mautic/0/rancher-compose.yml

```diff
@@ -0,0 +1,41 @@
+.catalog:
+  name: "Mautic"
+  version: "v1.0.0"
+  description: "Mautic provides free and open source marketing automation software available to everyone. Free email marketing software, lead management software and more."
+  uuid: Mautic-0
+  minimum_rancher_version: v0.51.0
+  questions:    
+
+    - variable: public_port
+      description: "public port to access the mautic site"
+      label: "PUBLIC_PORT"
+      required: true
+      default: "80"
+      type: "int"
+
+    - variable: mautic_db_host
```

Comment:
With this package config, mautic_db_host has fixed value `mauticdb`, not variable.